webpack.common.ts

```diff
@@ -20,11 +19,6 @@ const commonConfig: Configuration = {
     publicPath: "/static",
   },
   resolve: {
-    plugins: [
```

Comment:
Mener denne måtte til for å kunne importere ting med `@/<mappe>/<fil>`. Funker det bra uten?

Reply:
Derfor den var der ja 🤔 Det funker tilsynelatende helt fint nå både i dev og lokalt. Og bygg/tester fungerer jo også. Kan vi da konkludere med at det er greit, eller er det et spesifikt tilfelle jeg burde sjekke?

Reply:
Skulle tro alt er greit da ja 👍